src/webextension/list/popup/components/item-list-panel.js

```diff
@@ -23,8 +26,10 @@ export default function ItemListPanel({inputRef}) {
 
   return (
     <Panel>
-      <PanelHeader>
-        <ItemFilter inputRef={inputRef}/>
+      <PanelHeader className={styles.searchHeader}>
+        <Toolbar className={styles.filterBar}>
```

Comment:
I'd say that the `<PanelHeader/>` should *implicitly* work like a toolbar, so let's remove this line and modify the styling for the `<PanelHeader/>` to be more toolbar-y, if that's a word.